doc/contribute/index.rst

```diff
@@ -761,3 +761,24 @@ Code component README template
 ==============================
 
 .. literalinclude:: code_component_README
+
+
+Contribution Roles and Responsibilities
+***************************************
+
+The Zephyr project defines a development process workflow using GitHub
+**Issues** to track feature, enhancement, and bug reports together with GitHub
+**Pull Requests** (PRs) for submitting and reviewing changes.  Zephyr
+community members work together with `project working groups
```

Comment:
`project working groups`, this is not a list of working groups, but of (most of all) forums.. In general many of them are not "working groups" with any authority.
Miss-understanding that leads to situations where contributors are led to run in circles until they give up in frustration.

I think the easiest way forward would be just to remove this paragraph and leave only the paragraph below

Reply:
@aescolar, I believe your primary issue is using the concept of "working groups" so I'm rewording things a bit to just say "community" where to me it is implied that community means owners, maintainers, working groups, etc... I suspect that we will likely want to clarify that a bit more to formalize some aspects of the process for clarity reasons... particularly how it relates to PRs.